src/v2/view-builder/components/ToggleTextLink.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) 2020, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { _, View } from 'okta';
+
+export default View.extend({
+
+  /**
+   * View
+   * the link
+   */
+  link: null,
+
+  /**
+   * View
+   * adds any view under the link
+   */
+  linkInfo: null,
+
+  initialize() {
+    View.prototype.initialize.apply(this, arguments);
+    let link = _.resultCtx(this, 'link', this);
+    const linkInfo = _.resultCtx(this, 'footerInfo', this);
+    this.add(link);
```

Comment:
if linkInfo is not present then it is just like `Link` ?

Reply:
then it just adds a link, yes